lib/eeny-meeny/models/experiment.rb

```diff
@@ -5,7 +5,9 @@
 module EenyMeeny
   class Experiment
 
-    COOKIE_EXPERIMENT_ID_REGEX = /\Aeeny_meeny_(.+)_v\d+\z/.freeze
+    COOKIE_EXPERIMENT_ID_REGEX = Regexp.new(
+      "\\A#{EenyMeeny::Cookie::EXPERIMENT_PREFIX}(.+)_v(\\d+)\\z"
```

Comment:
:+1: 

To make this change we also need to add a `require` statement for the cookie model at the top of the file.

`require 'eeny-meeny/models/cookie'`

Reply:
Added : https://github.com/corthmann/eeny-meeny/pull/12/commits/eb91b637e0dccf358eafa045a220ecccc94a6e44